backend/dataall/modules/dataset_sharing/services/dataset_alarm_service.py

```diff
@@ -51,9 +49,9 @@ def trigger_revoke_table_sharing_failure_alarm(
             target_environment: Environment,
     ):
         log.info('Triggering share failure alarm...')
-        subject = f'ALARM: DATAALL Table {table.GlueTableName} Revoking LF permissions Failure Notification'
+        subject = f'Data.all Revoke LF permissions Failure for Table {table.GlueTableName}'[:100]
         message = f"""
-    You are receiving this email because your DATAALL {self.envname} environment in the {self.region} region has entered the ALARM state, because it failed to revoke Lake Formation permissions for table {table.GlueTableName} with Lake Formation.
+    You are receiving this email because your Data.all {self.envname} environment in the {self.region} region has entered the ALARM state, because it failed to revoke Lake Formation permissions for table {table.GlueTableName} with Lake Formation.
```

Comment:
lowercase data.all

Reply:
I just did a quick search with case type match across all the code. 
Data.all is clearly the most used option over data.all.
data.all is only used in logs in quicksight.py. Everywhere else, Data.all is used. To be consistent across most files, I prefer using Data.all

Reply:
agreed